src/platform_impl/macos/app_delegate.rs

```diff
@@ -58,6 +68,20 @@ declare_class!(
             // TODO: Notify every window that it will be destroyed, like done in iOS?
             AppState::internal_exit();
         }
+
+        #[method(applicationOpenUrls:)]
+        fn application_open_urls(&self, urls: Url) -> () {
```

Comment:
This method has to match that in [`icrate::Foundation::AppKit::NSApplicationDelegate::application_openURLs`](https://docs.rs/icrate/latest/icrate/AppKit/trait.NSApplicationDelegate.html#method.application_openURLs) exactly, otherwise it just plain won't work.

```suggestion
        #[method(application:openUrls:)]
        fn application_openURLs(&self, application: &NSApplication, urls: &NSArray<NSURL>) {
```